src/cli/Peers.ts

```diff
@@ -0,0 +1,387 @@
+import fs from 'fs';
+import commander from 'commander';
+import { PolykeyAgent, PeerInfo } from '../lib/Polykey';
+import {} from '../../tmp/hole-punch-test/publicserver';
+import { actionRunner, pkLogger, PKMessageType, determineNodePath } from '.';
+
+function makeAddPeerCommand() {
+  return new commander.Command('add')
+    .description('add a new peer to the store')
+    .option('-b, --base64 <base64>', 'decode the peer info from a base64 string')
```

Comment:
Are you using multibase and multihash for this sort of stuff? We are doing so in Matrix, and it would be cool to standardize our hashing and basing to the mb codecs from ipfs community.

Reply:
no, this is a utility function that comes with `protobufjs` that encodes/decodes a protobuf message to and from base64

Reply:
I am actually thinking of using multihash for making a peerId from the public key